content/v2/domains.markdown

```diff
@@ -52,6 +52,29 @@ List all domains for the account 1010 that have name matching "example":
           -H 'Accept: application/json' \
           https://api.dnsimple.com/v2/1010/domains?name_like=example
 
+### Sorting
+
+Sort records with the `sorting` param by specifing the field and the sorting policy: `asc` for ascending and `desc` for descending.
+Multiple fields are allowed and MUST be comma separated.
+
+Name | Description
+-----|------------
+`:id` | Sort domains by ID
+`:name` | Sort domains by name
+`:expires_on` | Sort domains by expiration date
+`:created_at` | Sort domains by creation timestamp
+`:updated_at` | Sort domains by last update timestamp
+
+The default sorting policy is by ascending `:name`.
```

Comment:
We should probably add something like _"which is the equivalent to sorting them alphabetically"_. (Even though these docs are for technical people _by ascending `name`_ sounds way more confusing by alphabetically).